### PR TITLE
30 b add tags to tests for minimal regression test scope - Fixes

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,6 +203,9 @@ $TEST_MODULE
 Any additional parameters to `robot` can be passed using the wrapper by giving
 them as the second and further arguments to the script.
 
+Any additional parameters to `robot` can be passed using the wrapper by giving
+them after a separator '--'.
+
```
Comment (krystian-hebel) [inline README.md:208]: Contradicting information, probably caused by rebase?

File: scripts/test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (krystian-hebel) [inline scripts/test.sh:1]: This file probably shouldn't be added.

**Review by krystian-hebel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (philipandag) [reply README.md]: Yeah, rebase error

File: scripts/test.sh (hunk 2 above)
Comment (philipandag) [reply scripts/test.sh]: Fixed by a rebase

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**